packages/components/psammead-story-promo/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 0.1.5-alpha.1 | [PR#513](https://github.com/bbc/psammead/pull/513) Add Link with hover, focus and visited states |
```

Comment:
I think the alpha version should be `0.2.0-alpha.1` since you are adding the link to the component in a backward compatible manner
```suggestion
| 0.2.0-alpha.1 | [PR#513](https://github.com/bbc/psammead/pull/513) Add Link with hover, focus and visited states |
```

Reply:
Well spotted 👍